webpage_info/getting-started/1. Devfile Ecosystem/Create, Share, and Consume Devfiles.adoc

```diff
@@ -0,0 +1,33 @@
+== *Create, Share and Consume Devfiles*
+:imagesdir: ../../../images
+'''''
+The following is an example of a devfile ecosystem that is framed within the context of an organization looking to standardize their development environment.  In the simplest case, developers can just consume the devfiles that are available from the public community registry.  If the organization sees the need to author and share their own devfiles internally, they will need a role based approach where Developers, Devfile Authors, and Registry Administrators can interact together.
+
+image::devfileEcosystem.svg[]
+
+
+=== *The Main Areas of the Devfile Ecosystem*
+=== *Create*
+
+A devfile author aka “stack provider” can be an individual or a group representing a runtime vendor.  This role requires sound knowledge of the supported runtime so devfiles can be created with the proper guidance on how to build and run applications.
+
+If a runtime stack is not available in the public registry, an organization can choose to develop their own and keep it private for their in-house development.
+
+=== *Share*
+
+The public community registry is managed by the community and hosted by Red Hat.  This is one way to share your devfile if there is a benefit to the greater community.
+
+If an organization wants to keep their own devfiles private but wishes to share with other departments, they can assign a Devfile Registry Administrator to deploy, maintain, and manage the contents of their private registry and the default list of devfile registries available in a given cluster.  With this set up, they can use devfiles from both the public and private registries, which gives them the flexibility of options.
```

Comment:
There's nothing wrong with the first sentence, so definitely feel free to keep it as is. However, it is a little on the long side. So you could divide it into smaller sentences, something like this: `If an organization wants to keep their own devfiles private but wishes to share with other departments, they can assign a Devfile Registry Administrator. The Devfile Registry Administrator deploys, maintains, and manages the contents of their private registry and the default list of devfile registries available in a given cluster.`

Reply:
Updated with some modifications.  De-capitalized the title and removed `Devfile` just to keep it consistent with the existing title